README.md

```diff
@@ -20,13 +21,14 @@ Kommunicate has a powerful human + chatbot hybrid platform, where you can integr
 
 [Kompose](https://dashboard.kommunicate.io/bots/bot-builder) is a Kommunicate’s bot builder that help you in building your own bot, a techie, non-techie, or a person who doesn’t have any idea about chatbots can also build the bot. Anyone can create a chatbot with the Kompose without any assistance.
 
-![Kompose-build](https://user-images.githubusercontent.com/38066371/87525821-a519f000-c6a7-11ea-90d6-97e8fab4d1b3.gif)
+![Kompose-build](https://user-images.githubusercontent.com/91717856/215320213-8f09b877-dc17-46b8-aac2-1caf4a0b5be4.gif)
```

Comment:
quality is not good, also please don't use terms like "Test 1", "Test 2". put some real project examples

Reply:
Okay will create new one and add it and will check the quality also